### DISCUSSION TITLE
Compatibility with 1.21

### DISCUSSION THREAD

**Discussion by troyere** [iPortalTeam/ImmersivePortalsMod#1567] [General]
Hello there ! First of all, I wanted to tell you all : this mod is freaking awesome 🤘😁

I was wondering, is the compatibility with 1.21 planned in the future ?

**Comment by SalTeyJek004** [comment]
1.21 hasnt even been out for a week, be patient
and this mod has been updating to each new major version since 1.16 so it probably will.

**Comment by Nick1st** [comment]
Probably, will take some time. Please refrain from asking.
